packages/api/src/controllers/revenueTypeController.js

```diff
@@ -148,7 +149,10 @@ const revenueTypeController = {
       const trx = await transaction.start(Model.knex());
       const { revenue_type_id } = req.params;
       const farm_id = req.headers.farm_id;
-      const data = { revenue_name: req.body.revenue_name };
+      const data = {
+        revenue_name: req.body.revenue_name,
+        custom_description: req.body.custom_description ?? undefined,
```

Comment:
Why `undefined`? It looks like our ORM / db is going to coerce that into an empty string (is that what you see in your local db?) so wouldn't it make more sense to set it to an empty string explicitly here?

And this is an API check, right? Since the frontend will always send this field even if it's just an empty string?

Reply:
Yeah so it stays what it is before. I don't really like how this endpoint is not just using the raw request body and removing additional properties but yah.  Frontend should not send an empty string I hope but yeah mostly API check.